releases/release-6.1.1.md

```diff
@@ -0,0 +1,179 @@
+    - fix the problem that there may be some obsolete data left in storage which cannot be deleted [#5659](https://github.com/pingcap/tiflash/issues/5659)
+    - Reduce unnecessary CPU usage in some edge cases [#5409](https://github.com/pingcap/tiflash/issues/5409)
+    - Fix a bug that TiFlash can not work in a cluster using ipv6 [#5247](https://github.com/pingcap/tiflash/issues/5247)
+    - Fix a panic issue in parallel aggregation when an exception is thrown. [#5356](https://github.com/pingcap/tiflash/issues/5356)
+
++ Tools
+
+    + TiCDC
+
+        - 修复版本兼容性问题 [#6039](https://github.com/pingcap/tiflow/issues/6039)
+        - 修复cdc server 启动未完成接受请求出现 panic 问题 [#5639](https://github.com/pingcap/tiflow/issues/5639)
+        - 修复 sync-point 打开时 ddl sink 可能出现 panic 的问题. [#4934](https://github.com/pingcap/tiflow/issues/4934)
+        - 修复 cdc server 重启时 API 工作不正常的问题. [#5837](https://github.com/pingcap/tiflow/issues/5837)
+        - 修复 black hole sink 场景下出现的 data race 问题. [#6206](https://github.com/pingcap/tiflow/issues/6206)
+        - 修复disable old value 时可能出现的 cdc panic 问题 。 [#6198](https://github.com/pingcap/tiflow/issues/6198)
+        - 修复在打开 redo 功能时，可能出现的一些数据不一致问题。[#6189](https://github.com/pingcap/tiflow/issues/6189) [#6368](https://github.com/pingcap/tiflow/issues/6368) [#6277](https://github.com/pingcap/tiflow/issues/6277) [#6456](https://github.com/pingcap/tiflow/issues/6456) [#6695](https://github.com/pingcap/tiflow/issues/6695) [#6764](https://github.com/pingcap/tiflow/issues/6764) [#6859](https://github.com/pingcap/tiflow/issues/6859)
+        - 采取异步写的方式提升 Redo 吞吐。 [#6011](https://github.com/pingcap/tiflow/issues/6011)
+        -修复打开 sync-point 功能在某些特殊场景下出现卡住 changefeed 的问题 。 [#6827](https://github.com/pingcap/tiflow/issues/6827)
+        - 修复 mysql sink 链接 ipv6 的问题。[#6135](https://github.com/pingcap/tiflow/issues/6135)
+
+    + Backup & Restore (BR)
+
+        (dup: release-5.4.2.md > Bug Fixes> Tools> Backup & Restore (BR))- 修复了 RawKV 模式下 BR 报 `ErrRestoreTableIDMismatch` 错误的问题 [#35279](https://github.com/pingcap/tidb/issues/35279)
+        (dup: release-6.2.0.md > 改进提升> Tools> Backup & Restore (BR))- 优化了全量备份数据组织形式，解决大规模集群备份时遇到的 S3 限流问题 [#30087](https://github.com/pingcap/tidb/issues/30087)
+        - 修复日志中统计备份耗时不正确的问题 [#35553](https://github.com/pingcap/tidb/issues/35553)
+
+    + Dumpling
+
+        - 使用 net.JoinHostPort 拼接 Host、Port，提供更好的 IPv6 兼容 [#36112](https://github.com/pingcap/tidb/issues/36112)
+
+    + TiDB Lightning
+
+        - 修复 IPv6 支持 [#35880](https://github.com/pingcap/tidb/issues/35880)
+        - 修复 read index not ready 问题，增加重试机制 [#36566](https://github.com/pingcap/tidb/issues/36566)
+        - 移除日志中的敏感信息 [#36374](https://github.com/pingcap/tidb/issues/36374) 
+        - 支持 Parquet 格式带`_`的特殊列名[#36980](https://github.com/pingcap/tidb/issues/36980)
+        - 修复极端情况下重复去重可能会 panic 的问题 [#36163](https://github.com/pingcap/tidb/issues/34163)
+
+    + TiDB Binlog
```

Comment:
fix drainer can't send request correctly to pump when set compressor=gzip [#1152](https://github.com/pingcap/tidb-binlog/issues/1152)